TrackletAlgorithm/SWLUTReader.h

```diff
@@ -0,0 +1,34 @@
+#include <fstream>
+#include <string>
+
+// Define a global mutex for locking multi-threading
```

Comment:
This comment "Define a global mutex" seems in the wrong place. Nothing is defined here.